test/functional/tests/fault_injection/test_standby_activate_the_same_cache_path.py

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+
+from api.cas import casadm
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_utils.size import Size, Unit
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.nand, DiskType.optane]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_standby_activate_the_same_cache_path():
+    with TestRun.step("Prepare a partition."):
+        cache = TestRun.disks["cache"]
+        cache.create_partitions([Size(200, Unit.MebiByte)])
+        cache_dev = cache.partitions[0]
+
+    with TestRun.step("Start a regular cache instance on the partition and stop it."):
+        cache = casadm.start_cache(cache_dev, force=True)
+        cache.stop()
+
+    with TestRun.step("Load standby cache on the partition with valid CAS metadata."):
+        cache = casadm.standby_load(cache_dev)
+
+    with TestRun.step("Detach standby cache"):
+        cache.standby_detach()
+
+    with TestRun.step("Activate cache providing path to the partition."):
+        cache.standby_activate(device=cache_dev)
```

Comment:
Consider adding check to determine if activate succeeded.
You can use cache statistics or check if exported object appeared properly.

Reply:
done